### PR TITLE
better validation in metadata matcher

### PR DESCRIPTION
**Pull request by prolic**

### CODE REVIEW

File: src/Metadata/MetadataMatcher.php [hunk 1]
```diff
@@ -35,12 +35,6 @@ public function withMetadataMatch(
             $fieldType = FieldType::METADATA();
         }
 
-        if ($fieldType->is(FieldType::MESSAGE_PROPERTY())
-            && ! in_array($field, ['uuid', 'createdAt', 'created_at', 'messageName', 'message_name'], true)
-        ) {
-            throw new InvalidArgumentException(sprintf('Invalid message property "%s" given', $field));
```
Comment (prolic) [inline src/Metadata/MetadataMatcher.php]: I needed to remove that, because the fields have different names in pdo-event-store tables

Comment (codeliner) [inline src/Metadata/MetadataMatcher.php]: but pdo-event-store can map them. I'd like to provide default filters with the event-store-mgmt-ui and I don't know which event-store implementation is used.

**Comment by prolic** [comment]
Problem is persistence strategy can name to fields like it wants to and
interface is final.

**Comment by codeliner** [comment]
ok then we need to find another way for the mgmt-ui

**Pull request merged**